### ISSUE TITLE
Houses endpoint responds with 404 on sold house

### ISSUE THREAD

**Issue by Krabba** [TibiaData/tibiadata-api-go#114]
**ONLY OCCURS DURING SERVER SAVE (10:00 CEST).**

If any house in the list includes "finished", the API will respond with a blank page, or 404.
Requests towards the API will get 500 back.

![image](https://user-images.githubusercontent.com/11424371/163981301-2766ea09-556d-4fbd-8c2a-912246a92d79.png)

Example links:
https://api.tibiadata.com/v3/houses/Antica/Ankrahmun
https://api.tibiadata.com/v3/houses/Nadora/Liberty%20Bay

**Comment by Krabba** [comment]
Examples of hosues that will be sold tomorrow **2022-04-20** 10:00 CEST (Ctrl + F for `hours`):
https://api.tibiadata.com/v3/houses/Antica/Venore
https://api.tibiadata.com/v3/houses/Antica/Yalahar
https://api.tibiadata.com/v3/houses/Ardera/Carlin
https://api.tibiadata.com/v3/houses/Havera/Darashia
https://api.tibiadata.com/v3/houses/Lutabra/Darashia
https://api.tibiadata.com/v3/houses/Solidera/Darashia
https://api.tibiadata.com/v3/houses/Xandebra/Carlin

**Comment by Krabba** [comment]
Example image of sold house:
![image](https://user-images.githubusercontent.com/11424371/164184808-749f5595-1fab-468b-ba2b-ed080766ead3.png)

Example string:
```Fibula Village, Villa
This house can have up to 7 beds.

The house has a size of 222 square meters. The monthly rent is 200k gold and will be debited to the bank account on Damora.

The house is currently being auctioned. The auction has ended at Apr 20 2022, 10:00:00 CEST. The highest bid so far is 170000 gold and has been submitted by Wytankuje Chyba Wszystko.

**Comment by tobiasehlert** [comment]
Thanks for reporting, this should be resolved with PR #115 👍
